examples/statemachine/test/generator.test.ts

```diff
@@ -0,0 +1,212 @@
+/******************************************************************************
+ * Copyright 2022 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+import { EmptyFileSystem, Generated, toString } from 'langium';
+import { parseHelper } from "langium/lib/test";
+import { generateCppContent } from '../src/cli/generator';
+import { Statemachine } from '../src/language-server/generated/ast';
+import { createStatemachineServices } from '../src/language-server/statemachine-module';
+
+describe('Tests the code generator', () => {
```

Comment:
The statemachine package.json is missing a `test` script.

Reply:
Right, I'll add one.

Reply:
Added!